### ISSUE TITLE
Add Support to validate  jsonPath in ExtractVariables policy

### ISSUE THREAD

**Issue by anaik91** [apigee/apigeelint#524]
Noticed a error when migrating from Apigee Edge to Apigee Hybrid. 

Below policy works fine in Edge , but not in Hybrid/X.
```
<ExtractVariables name="ExtractVariables-3">
  <Source>response</Source>
  <JSONPayload>
    <Variable name="latitude" type="float">
      <JSONPath>$.</JSONPath>
    </Variable>
  <VariablePrefix>geocoderesponse</VariablePrefix>
</ExtractVariables>
```

if you have  `$.` in `JSONPath` it fails at deployment.
The Api gets created/imported without any issues , but fails only while deploying.

```
Status: instance "xxxx" reported error steps.extractvariables.JSONPathCompilationFailed: "Failed to compile jsonpath $."
```
The ask is to catch this during linting phase.

**Comment by ssvaidyanathan** [comment]
@anaik91 - shouldnt this be reported for Edge too?

**Comment by DinoChiesa** [comment]
We will have to investigate further. The compilation check that is happening at deploy time, is using a Java-based JSONpath library.  The Apigeelint tool, as you know, is not Java based.  So we may not be able to replicate the exact results, unfortunately.

**Comment by DinoChiesa** [comment]
I think #535 addresses this issue.

**Comment by ssvaidyanathan** [comment]
Fixed in v2.67.0
